src/actions/render.js

```diff
@@ -3,21 +3,19 @@ const Handlebars = require("handlebars");
 const Result = require("../utils/result");
 const { normalizeYaml, PLURAL, SINGULAR } = require("../actions/normalize");
 const { loadFile } = require("../utils/fileUtils");
-const { accessiblityKeywords, groupKeywords, platformKeywords } = require("../model/keywords");
+const { accessiblityKeywords, groupKeywords, outputType } = require("../model/keywords");
 const { groupByKey } = require("../utils/arrayUtils");
 const { flatten } = require("../utils/arrayUtils");
 
 const percentEncodingPattern = /%(?!\d+{{.}})/;
 
-const render = (data, outputPath, platforms, languages) => {
-  const translations = normalizeYaml(data, platforms, languages);
+const render = (data, outputPath, languages, platforms, collections) => {
```

Comment:
Should we use outputTypes here instead of platforms? because I saw you've replaced this everywhere

Reply:
Good one!